src/client/common/installer/productInstaller.ts

```diff
@@ -158,20 +158,13 @@ export class DataScienceInstaller extends BaseInstaller {
     // Override base installer to support a more DS-friendly streamlined installation.
     public async install(
         product: Product,
-        interpreterUri?: InterpreterUri,
+        interpreter: PythonEnvironment,
         cancel?: CancellationToken,
         reInstallAndUpdate?: boolean,
         installPipIfRequired?: boolean
     ): Promise<InstallerResponse> {
-        // Precondition
-        if (isResource(interpreterUri)) {
```

Comment:
This would never happen with our code path, hence removed this, typings ensure this never happens.
I think this was a carry on from when we moved code from Python repo